### DISCUSSION TITLE
SQLite error on SELECT Documentation Error

### DISCUSSION THREAD

**Discussion by Shai2u** [pyproj4/pyproj#765] [General]
Hi,
I had a very annoying problem where on my Mac, Running Python I could not set or use CRS in GeoPandas in Jupyter however in Visual Studio Code it worked fine.
eventually I found out that I had two versions  of pyroj data, one which was flawed for some reason...

first of all I would like to mentioned that their was an error in the documentation:
https://pyproj4.github.io/pyproj/stable/gotchas.html#internal-proj-error-sqlite-error-on-select
it mentions the wrong method to set the data dir. - so it took me a week to figure that out.

however secondly since I am new in writing comments in GitHUB, I do not know how to participate in the Issue reports, I could not find an issue similar to mine and I did not know how to approach of any the contributor programmers,

so these are my two request/questions
1. to fix the doucmnation
2. how do I approach a contributor or take place in the Issues discussion?

**Comment by snowman2** [comment]
Welcome! This is the place to ask questions from the contributors.

If you would like to fix the documentation, you can submit a pull request through the GitHub website by going to: https://github.com/pyproj4/pyproj/blob/master/docs/gotchas.rst#internal-proj-error--sqlite-error-on-select, selecting the edit button at the top of the page, making the change, and saving the change at the bottom of the page.

To open an issue, you can go to: https://github.com/pyproj4/pyproj/issues
